tests/Functional/NewFilesOnlyEqualTimestampFunctionalTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
```

Comment:
kurňa, to je elaborátní test :D z toho zpětně dekódovat, co to mělo testovat, to je fuška teda. za tohle bych si nafackoval a nechal to stokrát přespat. teď to nechme pro budoucí generace :-)

Reply:
Oka, souhlasím, když bude chvilka tak se do dalšího kola rád pustím :)